dev/com.ibm.ws.session.db/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -75,6 +75,9 @@ use.invalidated.id.desc=Set this property to "true" to reuse the incoming identi
 use.oracle.blob=Use Binary Large Object (BLOB) for medium column
 use.oracle.blob.desc=Set this property to "true" to create the database table using the Binary Large Object (BLOB) data type for the medium column. This value increases performance of persistent sessions when Oracle databases are used. Due to an Oracle restriction, BLOB support requires use of the Oracle Call Interface (OCI) database driver for more than 4000 bytes of data. You must also ensure that a new sessions table is created before the server is restarted by dropping your old sessions table or by changing the datasource definition to reference a database that does not contain a sessions table.
 
+oracle.row.size.name=Oracle row size limit
+oracle.row.size.desc=Set megabyte-per-record limit. The default is 2-megabyte limit of stored data per record.
```

Comment:
suggested rewrite:
Set the limit of stored data per record, in megabytes. The default is 2 megabytes.

Reply:
Thanks for the rewrite.